Dockerfile

```diff
@@ -60,3 +60,5 @@ USER circleci
 
 # Define working directory
 WORKDIR /home/circleci
+
+RUN echo -e "Tag for this image:\n$(java -version 2>&1 | head -n 1 | grep -o '[[:digit:]._]*')-${SCALA_VERSION}-${KUBECTL_VERSION}-1"
```

Comment:
> include the kubectl version

Yes, definitely. That was an inconsistency between git tags (where it was missing) and Docker Hub tags. Since I recently figured out how to create automatic DockerHub builds from git tags _and use the same tag_, that needs to be aligned:
<img width="1090" alt="image" src="https://user-images.githubusercontent.com/7119122/63080483-7cdabc00-bf41-11e9-855b-6ff69f0c25fb.png">

However, I'd like to keep the "tag-versioning" consistent, so I would prefer to stick with `8u222` instead of `1.8.0_222` — that comes from the base image, `circleci/openjdk:8u222-jdk-stretch` which itself is based on `openjdk:8u222-jdk-stretch`.
Unfortunately, I didn't find a command that outputs this string 😔 
I would resort to changing it manually, _or_ put this logic into the CircleCi-Config and `grep` the `Dockerfile` to extract the name of the base image…

> include another version for the image that can be incremented if neither scala nor kubectl version changes

While I understand why this might help, it breaks consistency. And for users of the image, it doesn't really help, because the number doesn't tell you anything about _what_ changed.

```suggestion
RUN echo -e "Tag for this image:\n8u222-${SCALA_VERSION}-${KUBECTL_VERSION}"
```

Reply:
Hm but how would you tag a new image that has changes but none of the scala/kubectl/java versions changed? :thinking:

Reply:
Hm, `8u222-2.12.9-14.5-with-foo-fix`?